build.yaml

```diff
@@ -19,3 +20,10 @@ builders:
     builder_factories: ["buildPana"]
     build_extensions: {".dart": [".g.dart"]}
     build_to: source
+
+  version:
+    # See https://github.com/dart-lang/build/issues/1423
+    import: "../../../tool/builder.dart"
+    builder_factories: ["versionBuilder"]
+    build_extensions: {"$lib$": [".dart"]}
```

Comment:
This is sufficient because we only care (today) about whether this needs to run before other builders that require all Dart inputs... but it would be better to make it match the real build extensions

Reply:
Explain?

Reply:
the extensions indicate you're going to writ the file `lib/.dart` because they don't match the extension on your actual builder which are `{"$lib$", ["src/version.dart"]}`. It doesn't actually hurt based on how we currently use `build_extensions` from this config, but it's still odd.

Reply:
Fixed. I wish we had better – something – here. Like a warning if this doesn't align w/ the content in code?